pkgs/build-support/rust/default-crate-overrides.nix

```diff
@@ -357,4 +358,15 @@ in
     buildInputs = [ atk ];
   };
 
+  # Assumes it can run Command::new(env::var("CARGO")).arg("locate-project")
+  # https://github.com/bkchr/proc-macro-crate/blame/master/src/lib.rs#L244
+  proc-macro-crate = attrs: lib.optionalAttrs (lib.versionAtLeast attrs.version "2.0") {
+    prePatch = (attrs.prePatch or "") + ''
```

Comment:
Why is that not postPatch? This could break patches being applied to the source.

Reply:
Ask the author directly please, he's not using GitHub anymore: https://lists.sr.ht/~andir/nixpkgs-dev/patches/54642

Reply:
So we both don't know it, so I assume this was just an oversight. #340790

Also if people try to avoid easy communication, I am not in favor of chasing after them.